its/tests/src/test/java/its/SonarQubeDeveloperEditionTests.java

```diff
@@ -1212,8 +1207,14 @@ void shouldContainExtendedDescription() throws Exception {
       await().untilAsserted(() -> assertThat(didSynchronizeConfigurationScopes).isNotEmpty());
 
       var ruleDetailsResponse = backend.getRulesService().getEffectiveRuleDetails(new GetEffectiveRuleDetailsParams(CONFIG_SCOPE_ID, javaRuleKey("S106"), null)).get();
-      var ruleTabs = ruleDetailsResponse.details().getDescription().getRight().getTabs();
-      assertThat(ruleTabs.get(ruleTabs.size() -1).getContent().getLeft().getHtmlContent()).contains(expected);
+      var ruleDescription = ruleDetailsResponse.details().getDescription();
+      if (!ORCHESTRATOR.getServer().version().isGreaterThanOrEquals(9, 5)) {
```

Comment:
Minor comment, but I find this kind of double negation difficult to read. Maybe invert the if/else condition?

Reply:
Addressed in 48b21ba